ukwa/npld-player
# Setup Electron app

(https://github.com/ukwa/npld-player/issues/5) Scaffolds [Electron](https://www.electronjs.org/) + TypeScript app using [Electron Forge](https://www.electronforge.io/).

### Manual test

Follow quickstart instructions in README: `yarn && yarn start`. Verify that Electron app starts up.

### Screenshots

On OSX 12.1:
<img width="912" alt="Screen Shot 2022-04-28 at 11 52 02 AM" src="https://user-images.githubusercontent.com/4672952/165825607-ece5a288-1b40-409a-b769-8bed70e7edbe.png">
